### ISSUE TITLE
Labeled data available?

### ISSUE THREAD

**Issue by Jordan-Pierce** [josauder/mee-deepreefmap#2]
Hi, this a great paper! Are the labels available along with the original training data?

**Comment by sgraine** [comment]
I would also love the know the answer to this!

This zenodo link seems to have some videos available but not the polygon annotations described in the paper: https://zenodo.org/records/10624794.

Would be awesome to access the original labels used in the paper.

**Comment by pingchuan** [comment]
> 我也想知道这个问题的答案！
> 
> 这个 zenodo 链接似乎有一些可用的视频，但没有论文中描述的多边形注释：https://zenodo.org/records/10624794。
> 
> 访问论文中使用的原始标签会很棒。

I'm a newbie, I found that the lossfunction function of the sfm folder is missing, can you please complete it?

**Comment by josauder** [comment]
Hi! Sorry about this! Will update the Zenodo repository as soon as possible!

**Comment by MichaelRamamonjisoa** [comment]
Hi, paper results seem great! I was wondering if you're planning to release the full dataset some time?
